src/main/scala/com/elarib/PartialSbtPlugin.scala

```diff
@@ -2,12 +2,13 @@ package com.elarib
 
 import com.elarib.model.{ChangeGetter, PartialSbParser}
 import org.apache.logging.log4j.LogManager
-import sbt.Keys._
+import sbt.*
+import sbt.Keys.*
```

Comment:
WDYT about keeping the old-style for wildcard imports?

Reply:
IDEA is constantly bugging me with this. There's `-Xsource:3` flag somewhere I could not find.

Reply:
Done